Meta/Lagom/CMakeLists.txt

```diff
@@ -657,6 +657,15 @@ if (BUILD_LAGOM)
                 WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/../../Tests/LibVideo)
         endforeach()
 
+        # Web
+        if (ENABLE_LAGOM_LIBWEB)
+            file(GLOB LIBWEB_TEST_SOURCES CONFIGURE_DEPENDS "../../Tests/LibWeb/*.cpp")
+            foreach(source ${LIBWEB_TEST_SOURCES})
+                lagom_test(${source} LIBS LibWeb
+                    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/../../Tests/LibWeb)
+            endforeach()
+        endif()
```

Comment:
I think we do prefer not globbing (like the js tests below) but we also do it above here so :shrug:

Reply:
I decided against removing globbing, because we do that everywhere and isn't it nicer when one doesn't have to manually add a new Test file? ^^